tests/test_umath.py

```diff
@@ -279,6 +279,19 @@ def test_invalid_shape(self, shape):
             dpnp.copysign(dp_array, dp_array, out=dp_out)
 
 
+class TestDegrees:
+    @pytest.mark.parametrize(
+        "dtype", get_all_dtypes(no_none=True, no_complex=True)
+    )
+    def test_degrees(self, dtype):
+        a = numpy.array([numpy.pi, -0.5 * numpy.pi], dtype=dtype)
```

Comment:
Could you cover more cases for this function?
For example, `numpy.array([0])`, array with `nan` and `+-inf` and result with `large angles`

Reply:
`0` value is covered by `tests/test_usm_type.py::test_1in_1out` and `tests/test_sycl_queue.py::test_1in_1out` tests

Reply:
More tests are added